src/posit/connect/external/databricks.py

```diff
@@ -3,7 +3,7 @@
 from typing import Callable, Dict, Optional
 
 from ..client import Client
-from ..oauth import OAuthIntegration
+from ..oauth.oauth import OAuth
```

Comment:
Sorry this is going to conflict with my in flight PR (#268 ) but it should be trivial to resolve.

Reply:
nice - yeah if you want to merge first I'm happy to sort out the conflict on my branch. I want to still do a bit of manual testing with these new resources to figure out some cookbook examples, so no need to merge this immediately.